<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateCourse</class>
 <widget class="QDialog" name="CreateCourse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>301</width>
     <height>338</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>课程名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="course_cname_edit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>其余教师</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QListWidget" name="course_cteacher_list"/>
        </item>
        <item>
         <widget class="QPushButton" name="course_add_teacher_btn">
          <property name="text">
           <string>添加教师</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>课程简介</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="course_cintro_text"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>所属院系</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="course_cdept_combo"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>所属专业</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="course_cmajor_combo"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="course_create_btn">
        <property name="text">
         <string>开设课程</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="course_cancel_btn">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
